dev/com.ibm.ws.jaxws.2.3.web/bnd.bnd

```diff
@@ -0,0 +1,83 @@
+#*******************************************************************************
+# Copyright (c) 2019 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+
+bVersion=1.0
+
+Bundle-Name: JAX-WS 2.3 Web
+Bundle-SymbolicName: com.ibm.ws.jaxws.2.3.web
+Bundle-Description: IBM JAX-WS 2.3 Web support; version=${bVersion}
+
+-dsannotations: com.ibm.ws.jaxws.web.WebXMLEndpointInfoConfigurator
+
+Service-Component: \
+  com.ibm.ws.jaxws.web.class.list.collaborator; \
+     implementation:=com.ibm.ws.jaxws23.web.JaxWsInjectionClassListCollaborator ; \
+     provide:='com.ibm.wsspi.webcontainer.collaborator.WebAppInjectionClassListCollaborator'; \
+     immediate:=true; \
+     properties:="service.vendor=IBM", \
+  com.ibm.ws.jaxws.web.jaxws.module.info.builder; \
+     implementation:=com.ibm.ws.jaxws23.web.WebJaxWsModuleInfoBuilder; \
+     provide:='com.ibm.ws.jaxws.metadata.builder.JaxWsModuleInfoBuilder'; \
+     endpointInfoBuilder=com.ibm.ws.jaxws.metadata.builder.EndpointInfoBuilder; \
+     jaxWsModuleInfoBuilderExtension=com.ibm.ws.jaxws.metadata.builder.JaxWsModuleInfoBuilderExtension*; \
+     dynamic:='endpointInfoBuilder'; \
+     optional:='endpointInfoBuilder'; \
+     configuration-policy:=optional; \
+     activate:=activate; \
+     deactivate:=deactivate; \
+     properties:="service.vendor=IBM", \
+  com.ibm.ws.jaxws.web.jaxwsServletEndpointConfigurator; \
+     implementation:=com.ibm.ws.jaxws23.web.JaxWsServletEndpointConfigurator; \
+     provide:='com.ibm.ws.jaxws.endpoint.JaxWsEndpointConfigurator'; \
+     properties:="service.vendor=IBM"
+
+Export-Package: \
+    com.ibm.ws.jaxws23.web
+
+Import-Package: \
+   com.ibm.wsspi.adaptable.module.adapters, \
+   com.ibm.wsspi.webcontainer.collaborator, \
+   com.ibm.ws.container.service.metadata, \
+   javax.xml.ws.*;version="[2.2,3)", \
```

Comment:
Update javax.xml.ws.* and org.apache.cxf.* to 2.3 and 3.3 respectively

Reply:
I decided to update these in a future PR after the project has been created and we start using it